src/compute-client/src/protocol/command.rs

```diff
@@ -339,6 +339,8 @@ pub struct ComputeParameters {
     pub tracing: TracingParameters,
     /// gRPC client configuration.
     pub grpc_client: GrpcClientParameters,
+    /// Enable arrangement size logging
+    pub enable_arrangement_size_logging: Option<bool>,
```

Comment:
Nit: Can you move that up to `enable_mz_join_core`, to keep the feature flags close together?